examples/newrelic-express-apm/src/utils/response.js

```diff
@@ -52,6 +58,11 @@ class ResponseHandler {
       responseBody.data = data
     }
 
+    agent.recordMetric(
+      `Commercetools/Client/Response/Error/${statusCode}`,
+      statusCode
```

Comment:
Should this be a counter metric instead? Same for success metric above.

Reply:
No, it shouldn't, take a look at [this docs](https://docs.newrelic.com/docs/apm/agents/nodejs-agent/extend-your-instrumentation/nodejs-custom-metrics/#example) maybe you will understand why it should be `recordMetric` method. Also, It is important that you know that there are different method implementation for the agents for individual languages.